cli/src/wallet.tsx

```diff
@@ -37,19 +42,16 @@ interface WalletContextProps {
   setWallet: (wallet: Wallet) => void
 }
 
-const loadDefaultWallet = () => {
-  if (!defaultWalletPrivateKey) {
-    return EmptyWallet
-  }
-  const a = tryPrivateKeyToAccount(getPk())
+const loadDefaultWallet = (pk: Hex) => {
```

Comment:
Aside from it is meant to load **default** wallet, a major reason this function does not take argument and does not rely on information inside React / hook is to provide an example of letting React/hook components access non-React/hook values (such as command line options from yarg)